robottelo/ui/computeresource.py

```
@@ -1,72 +1,116 @@
-        self.wait_for_ajax()
+            Select(
+                self.find_element(locators["resource.provider_type"])
+            ).select_by_visible_text(provider_type)
+        for parameter_name, parameter_value, parameter_type in parameter_list:
+            param_locator = '.'.join((
+                'resource',
+                (parameter_name.lower()).replace(' ', '_')
+            ))
+            self.wait_until_element(locators[param_locator])
+            if parameter_type == 'field':
+                self.find_element(
+                    locators[param_locator]).send_keys(parameter_value)
+            elif parameter_type == 'select':
+                Select(
+                    self.find_element(locators[param_locator])
+                ).select_by_visible_text(parameter_value)
+            elif parameter_type == 'checkbox':
+                if (self.find_element(locators[param_locator]).is_selected() !=
+                        parameter_value):
+                    self.click(locators[param_locator])
+            elif parameter_type == 'special select':
+                button_locator = '.'.join((
+                    'resource',
+                    (parameter_name.lower()).replace(' ', '_'),
+                    'button'
+                ))
+                self.click(locators[button_locator])
+                self.find_element(
+                    locators[param_locator]
+                ).select_by_visible_text(parameter_value)
+
+    def _configure_orgs(self, orgs, org_select):
+        """Provides configuration capabilities for compute resource
+        organization. The following format should be used::
 
-    def create(self, name, orgs, description=None, org_select=True,
-               provider_type=None, url=None, user=None,
-               password=None, region=None, libvirt_display=None,
-               libvirt_set_passwd=True, tenant=None):
+            orgs=['Aoes6V', 'JIFNPC'], org_select=True
+
+        """
+        if orgs:
```

Should this raise an exception if orgs is not provided?